CRITICAL - 2022-02-19 00:05:55 --> Undefined variable $request
#0 /var/www/html/ci418/app/Controllers/Auth.php(106): CodeIgniter\Debug\Exceptions->errorHandler()
#1 /var/www/html/ci418/system/CodeIgniter.php(825): App\Controllers\Auth->verifySign()
#2 /var/www/html/ci418/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#3 /var/www/html/ci418/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#4 /var/www/html/ci418/public/index.php(38): CodeIgniter\CodeIgniter->run()
#5 /var/www/html/ci418/system/Commands/Server/rewrite.php(43): require_once('...')
#6 {main}
CRITICAL - 2022-02-19 00:17:03 --> There is no data to update.
#0 /var/www/html/ci418/system/BaseModel.php(1574): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset()
#1 /var/www/html/ci418/system/BaseModel.php(832): CodeIgniter\BaseModel->transformDataToArray()
#2 /var/www/html/ci418/system/Model.php(651): CodeIgniter\BaseModel->update()
#3 /var/www/html/ci418/app/Controllers/Auth.php(115): CodeIgniter\Model->update()
#4 /var/www/html/ci418/system/CodeIgniter.php(825): App\Controllers\Auth->verifySign()
#5 /var/www/html/ci418/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#6 /var/www/html/ci418/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#7 /var/www/html/ci418/public/index.php(38): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/ci418/system/Commands/Server/rewrite.php(43): require_once('...')
#9 {main}
CRITICAL - 2022-02-19 00:17:24 --> There is no data to update.
#0 /var/www/html/ci418/system/BaseModel.php(1574): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset()
#1 /var/www/html/ci418/system/BaseModel.php(832): CodeIgniter\BaseModel->transformDataToArray()
#2 /var/www/html/ci418/system/Model.php(651): CodeIgniter\BaseModel->update()
#3 /var/www/html/ci418/app/Controllers/Auth.php(115): CodeIgniter\Model->update()
#4 /var/www/html/ci418/system/CodeIgniter.php(825): App\Controllers\Auth->verifySign()
#5 /var/www/html/ci418/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#6 /var/www/html/ci418/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#7 /var/www/html/ci418/public/index.php(38): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/ci418/system/Commands/Server/rewrite.php(43): require_once('...')
#9 {main}
CRITICAL - 2022-02-19 00:17:44 --> Undefined variable $request
#0 /var/www/html/ci418/app/Controllers/Auth.php(106): CodeIgniter\Debug\Exceptions->errorHandler()
#1 /var/www/html/ci418/system/CodeIgniter.php(825): App\Controllers\Auth->verifySign()
#2 /var/www/html/ci418/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#3 /var/www/html/ci418/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#4 /var/www/html/ci418/public/index.php(38): CodeIgniter\CodeIgniter->run()
#5 /var/www/html/ci418/system/Commands/Server/rewrite.php(43): require_once('...')
#6 {main}
CRITICAL - 2022-02-19 00:17:56 --> There is no data to update.
#0 /var/www/html/ci418/system/BaseModel.php(1574): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset()
#1 /var/www/html/ci418/system/BaseModel.php(832): CodeIgniter\BaseModel->transformDataToArray()
#2 /var/www/html/ci418/system/Model.php(651): CodeIgniter\BaseModel->update()
#3 /var/www/html/ci418/app/Controllers/Auth.php(116): CodeIgniter\Model->update()
#4 /var/www/html/ci418/system/CodeIgniter.php(825): App\Controllers\Auth->verifySign()
#5 /var/www/html/ci418/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#6 /var/www/html/ci418/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#7 /var/www/html/ci418/public/index.php(38): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/ci418/system/Commands/Server/rewrite.php(43): require_once('...')
#9 {main}
CRITICAL - 2022-02-19 00:21:49 --> Call to undefined function App\Controllers\request()
#0 /var/www/html/ci418/system/CodeIgniter.php(825): App\Controllers\Auth->verifySign()
#1 /var/www/html/ci418/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#2 /var/www/html/ci418/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#3 /var/www/html/ci418/public/index.php(38): CodeIgniter\CodeIgniter->run()
#4 /var/www/html/ci418/system/Commands/Server/rewrite.php(43): require_once('...')
#5 {main}
ERROR - 2022-02-19 00:22:13 --> Unknown column '$email' in 'where clause'
CRITICAL - 2022-02-19 00:22:13 --> Unknown column '$email' in 'where clause'
#0 /var/www/html/ci418/system/Database/MySQLi/Connection.php(292): mysqli->query()
#1 /var/www/html/ci418/system/Database/BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute()
#2 /var/www/html/ci418/system/Database/BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 /var/www/html/ci418/app/Controllers/Auth.php(112): CodeIgniter\Database\BaseConnection->query()
#4 /var/www/html/ci418/system/CodeIgniter.php(825): App\Controllers\Auth->verifySign()
#5 /var/www/html/ci418/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#6 /var/www/html/ci418/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#7 /var/www/html/ci418/public/index.php(38): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/ci418/system/Commands/Server/rewrite.php(43): require_once('...')
#9 {main}
CRITICAL - 2022-02-19 00:25:00 --> There is no data to update.
#0 /var/www/html/ci418/system/BaseModel.php(1574): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset()
#1 /var/www/html/ci418/system/BaseModel.php(832): CodeIgniter\BaseModel->transformDataToArray()
#2 /var/www/html/ci418/system/Model.php(651): CodeIgniter\BaseModel->update()
#3 /var/www/html/ci418/app/Controllers/Auth.php(116): CodeIgniter\Model->update()
#4 /var/www/html/ci418/system/CodeIgniter.php(825): App\Controllers\Auth->verifySign()
#5 /var/www/html/ci418/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#6 /var/www/html/ci418/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#7 /var/www/html/ci418/public/index.php(38): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/ci418/system/Commands/Server/rewrite.php(43): require_once('...')
#9 {main}
CRITICAL - 2022-02-19 01:10:30 --> Too few arguments to function App\Controllers\Auth::verifySign(), 0 passed in /var/www/html/ci418/system/CodeIgniter.php on line 825 and exactly 1 expected
#0 /var/www/html/ci418/system/CodeIgniter.php(825): App\Controllers\Auth->verifySign()
#1 /var/www/html/ci418/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#2 /var/www/html/ci418/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#3 /var/www/html/ci418/public/index.php(38): CodeIgniter\CodeIgniter->run()
#4 /var/www/html/ci418/system/Commands/Server/rewrite.php(43): require_once('...')
#5 {main}
CRITICAL - 2022-02-19 01:10:56 --> Too few arguments to function App\Controllers\Auth::verifySign(), 0 passed in /var/www/html/ci418/system/CodeIgniter.php on line 825 and exactly 1 expected
#0 /var/www/html/ci418/system/CodeIgniter.php(825): App\Controllers\Auth->verifySign()
#1 /var/www/html/ci418/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#2 /var/www/html/ci418/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#3 /var/www/html/ci418/public/index.php(38): CodeIgniter\CodeIgniter->run()
#4 /var/www/html/ci418/system/Commands/Server/rewrite.php(43): require_once('...')
#5 {main}
CRITICAL - 2022-02-19 01:11:18 --> There is no data to update.
#0 /var/www/html/ci418/system/BaseModel.php(1574): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset()
#1 /var/www/html/ci418/system/BaseModel.php(832): CodeIgniter\BaseModel->transformDataToArray()
#2 /var/www/html/ci418/system/Model.php(651): CodeIgniter\BaseModel->update()
#3 /var/www/html/ci418/app/Controllers/Auth.php(118): CodeIgniter\Model->update()
#4 /var/www/html/ci418/system/CodeIgniter.php(825): App\Controllers\Auth->verifySign()
#5 /var/www/html/ci418/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#6 /var/www/html/ci418/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#7 /var/www/html/ci418/public/index.php(38): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/ci418/system/Commands/Server/rewrite.php(43): require_once('...')
#9 {main}
CRITICAL - 2022-02-19 01:15:31 --> There is no data to update.
#0 /var/www/html/ci418/system/BaseModel.php(1574): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset()
#1 /var/www/html/ci418/system/BaseModel.php(832): CodeIgniter\BaseModel->transformDataToArray()
#2 /var/www/html/ci418/system/Model.php(651): CodeIgniter\BaseModel->update()
#3 /var/www/html/ci418/app/Controllers/Auth.php(117): CodeIgniter\Model->update()
#4 /var/www/html/ci418/system/CodeIgniter.php(825): App\Controllers\Auth->verifySign()
#5 /var/www/html/ci418/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#6 /var/www/html/ci418/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#7 /var/www/html/ci418/public/index.php(38): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/ci418/system/Commands/Server/rewrite.php(43): require_once('...')
#9 {main}
CRITICAL - 2022-02-19 01:19:27 --> Cannot use object of type CodeIgniter\Database\MySQLi\Result as array
#0 /var/www/html/ci418/system/CodeIgniter.php(825): App\Controllers\Auth->verifySign()
#1 /var/www/html/ci418/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#2 /var/www/html/ci418/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#3 /var/www/html/ci418/public/index.php(38): CodeIgniter\CodeIgniter->run()
#4 /var/www/html/ci418/system/Commands/Server/rewrite.php(43): require_once('...')
#5 {main}
CRITICAL - 2022-02-19 01:20:04 --> Cannot use object of type CodeIgniter\Database\MySQLi\Result as array
#0 /var/www/html/ci418/system/CodeIgniter.php(825): App\Controllers\Auth->verifySign()
#1 /var/www/html/ci418/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#2 /var/www/html/ci418/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#3 /var/www/html/ci418/public/index.php(38): CodeIgniter\CodeIgniter->run()
#4 /var/www/html/ci418/system/Commands/Server/rewrite.php(43): require_once('...')
#5 {main}
CRITICAL - 2022-02-19 01:20:05 --> Cannot use object of type CodeIgniter\Database\MySQLi\Result as array
#0 /var/www/html/ci418/system/CodeIgniter.php(825): App\Controllers\Auth->verifySign()
#1 /var/www/html/ci418/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#2 /var/www/html/ci418/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#3 /var/www/html/ci418/public/index.php(38): CodeIgniter\CodeIgniter->run()
#4 /var/www/html/ci418/system/Commands/Server/rewrite.php(43): require_once('...')
#5 {main}
CRITICAL - 2022-02-19 01:20:18 --> Cannot use object of type CodeIgniter\Database\MySQLi\Result as array
#0 /var/www/html/ci418/system/CodeIgniter.php(825): App\Controllers\Auth->verifySign()
#1 /var/www/html/ci418/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#2 /var/www/html/ci418/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#3 /var/www/html/ci418/public/index.php(38): CodeIgniter\CodeIgniter->run()
#4 /var/www/html/ci418/system/Commands/Server/rewrite.php(43): require_once('...')
#5 {main}
CRITICAL - 2022-02-19 01:20:27 --> Cannot use object of type CodeIgniter\Database\MySQLi\Result as array
#0 /var/www/html/ci418/system/CodeIgniter.php(825): App\Controllers\Auth->verifySign()
#1 /var/www/html/ci418/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#2 /var/www/html/ci418/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#3 /var/www/html/ci418/public/index.php(38): CodeIgniter\CodeIgniter->run()
#4 /var/www/html/ci418/system/Commands/Server/rewrite.php(43): require_once('...')
#5 {main}
CRITICAL - 2022-02-19 01:20:41 --> Undefined property: CodeIgniter\Database\MySQLi\Result::$id
#0 /var/www/html/ci418/app/Controllers/Auth.php(113): CodeIgniter\Debug\Exceptions->errorHandler()
#1 /var/www/html/ci418/system/CodeIgniter.php(825): App\Controllers\Auth->verifySign()
#2 /var/www/html/ci418/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#3 /var/www/html/ci418/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#4 /var/www/html/ci418/public/index.php(38): CodeIgniter\CodeIgniter->run()
#5 /var/www/html/ci418/system/Commands/Server/rewrite.php(43): require_once('...')
#6 {main}
CRITICAL - 2022-02-19 01:21:02 --> Undefined property: CodeIgniter\Database\MySQLi\Result::$id
#0 /var/www/html/ci418/app/Controllers/Auth.php(113): CodeIgniter\Debug\Exceptions->errorHandler()
#1 /var/www/html/ci418/system/CodeIgniter.php(825): App\Controllers\Auth->verifySign()
#2 /var/www/html/ci418/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#3 /var/www/html/ci418/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#4 /var/www/html/ci418/public/index.php(38): CodeIgniter\CodeIgniter->run()
#5 /var/www/html/ci418/system/Commands/Server/rewrite.php(43): require_once('...')
#6 {main}
CRITICAL - 2022-02-19 01:21:42 --> Cannot use object of type CodeIgniter\Database\MySQLi\Result as array
#0 /var/www/html/ci418/system/CodeIgniter.php(825): App\Controllers\Auth->verifySign()
#1 /var/www/html/ci418/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#2 /var/www/html/ci418/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#3 /var/www/html/ci418/public/index.php(38): CodeIgniter\CodeIgniter->run()
#4 /var/www/html/ci418/system/Commands/Server/rewrite.php(43): require_once('...')
#5 {main}
CRITICAL - 2022-02-19 01:21:52 --> syntax error, unexpected variable "$query", expecting "," or ";"
#0 /var/www/html/ci418/system/Router/Router.php(175): CodeIgniter\Router\Router->autoRoute()
#1 /var/www/html/ci418/system/CodeIgniter.php(719): CodeIgniter\Router\Router->handle()
#2 /var/www/html/ci418/system/CodeIgniter.php(361): CodeIgniter\CodeIgniter->tryToRouteIt()
#3 /var/www/html/ci418/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#4 /var/www/html/ci418/public/index.php(38): CodeIgniter\CodeIgniter->run()
#5 /var/www/html/ci418/system/Commands/Server/rewrite.php(43): require_once('...')
#6 {main}
CRITICAL - 2022-02-19 01:22:22 --> Call to undefined method CodeIgniter\Database\MySQLi\Result::findAll()
#0 /var/www/html/ci418/system/CodeIgniter.php(825): App\Controllers\Auth->verifySign()
#1 /var/www/html/ci418/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#2 /var/www/html/ci418/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#3 /var/www/html/ci418/public/index.php(38): CodeIgniter\CodeIgniter->run()
#4 /var/www/html/ci418/system/Commands/Server/rewrite.php(43): require_once('...')
#5 {main}
CRITICAL - 2022-02-19 01:23:54 --> Undefined array key "id"
#0 /var/www/html/ci418/app/Controllers/Auth.php(115): CodeIgniter\Debug\Exceptions->errorHandler()
#1 /var/www/html/ci418/system/CodeIgniter.php(825): App\Controllers\Auth->verifySign()
#2 /var/www/html/ci418/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#3 /var/www/html/ci418/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#4 /var/www/html/ci418/public/index.php(38): CodeIgniter\CodeIgniter->run()
#5 /var/www/html/ci418/system/Commands/Server/rewrite.php(43): require_once('...')
#6 {main}
CRITICAL - 2022-02-19 01:24:21 --> Attempt to read property "id" on array
#0 /var/www/html/ci418/app/Controllers/Auth.php(115): CodeIgniter\Debug\Exceptions->errorHandler()
#1 /var/www/html/ci418/system/CodeIgniter.php(825): App\Controllers\Auth->verifySign()
#2 /var/www/html/ci418/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#3 /var/www/html/ci418/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#4 /var/www/html/ci418/public/index.php(38): CodeIgniter\CodeIgniter->run()
#5 /var/www/html/ci418/system/Commands/Server/rewrite.php(43): require_once('...')
#6 {main}
CRITICAL - 2022-02-19 01:24:57 --> Undefined array key "id"
#0 /var/www/html/ci418/app/Controllers/Auth.php(115): CodeIgniter\Debug\Exceptions->errorHandler()
#1 /var/www/html/ci418/system/CodeIgniter.php(825): App\Controllers\Auth->verifySign()
#2 /var/www/html/ci418/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#3 /var/www/html/ci418/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#4 /var/www/html/ci418/public/index.php(38): CodeIgniter\CodeIgniter->run()
#5 /var/www/html/ci418/system/Commands/Server/rewrite.php(43): require_once('...')
#6 {main}
CRITICAL - 2022-02-19 01:25:12 --> Attempt to read property "id" on array
#0 /var/www/html/ci418/app/Controllers/Auth.php(115): CodeIgniter\Debug\Exceptions->errorHandler()
#1 /var/www/html/ci418/system/CodeIgniter.php(825): App\Controllers\Auth->verifySign()
#2 /var/www/html/ci418/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#3 /var/www/html/ci418/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#4 /var/www/html/ci418/public/index.php(38): CodeIgniter\CodeIgniter->run()
#5 /var/www/html/ci418/system/Commands/Server/rewrite.php(43): require_once('...')
#6 {main}
CRITICAL - 2022-02-19 01:27:49 --> Undefined array key "id"
#0 /var/www/html/ci418/app/Controllers/Auth.php(121): CodeIgniter\Debug\Exceptions->errorHandler()
#1 /var/www/html/ci418/system/CodeIgniter.php(825): App\Controllers\Auth->verifySign()
#2 /var/www/html/ci418/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#3 /var/www/html/ci418/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#4 /var/www/html/ci418/public/index.php(38): CodeIgniter\CodeIgniter->run()
#5 /var/www/html/ci418/system/Commands/Server/rewrite.php(43): require_once('...')
#6 {main}
CRITICAL - 2022-02-19 01:31:07 --> CodeIgniter\BaseModel::transformDataToArray(): Return value must be of type array, string returned
#0 /var/www/html/ci418/system/BaseModel.php(704): CodeIgniter\BaseModel->transformDataToArray()
#1 /var/www/html/ci418/system/Model.php(625): CodeIgniter\BaseModel->insert()
#2 /var/www/html/ci418/system/BaseModel.php(658): CodeIgniter\Model->insert()
#3 /var/www/html/ci418/app/Controllers/Auth.php(121): CodeIgniter\BaseModel->save()
#4 /var/www/html/ci418/system/CodeIgniter.php(825): App\Controllers\Auth->verifySign()
#5 /var/www/html/ci418/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#6 /var/www/html/ci418/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#7 /var/www/html/ci418/public/index.php(38): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/ci418/system/Commands/Server/rewrite.php(43): require_once('...')
#9 {main}
CRITICAL - 2022-02-19 01:37:54 --> Attempt to read property "id" on array
#0 /var/www/html/ci418/app/Controllers/Auth.php(115): CodeIgniter\Debug\Exceptions->errorHandler()
#1 /var/www/html/ci418/system/CodeIgniter.php(825): App\Controllers\Auth->verifySign()
#2 /var/www/html/ci418/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#3 /var/www/html/ci418/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#4 /var/www/html/ci418/public/index.php(38): CodeIgniter\CodeIgniter->run()
#5 /var/www/html/ci418/system/Commands/Server/rewrite.php(43): require_once('...')
#6 {main}
CRITICAL - 2022-02-19 01:40:28 --> CodeIgniter\BaseModel::transformDataToArray(): Return value must be of type array, string returned
#0 /var/www/html/ci418/system/BaseModel.php(704): CodeIgniter\BaseModel->transformDataToArray()
#1 /var/www/html/ci418/system/Model.php(625): CodeIgniter\BaseModel->insert()
#2 /var/www/html/ci418/system/BaseModel.php(658): CodeIgniter\Model->insert()
#3 /var/www/html/ci418/app/Controllers/Auth.php(121): CodeIgniter\BaseModel->save()
#4 /var/www/html/ci418/system/CodeIgniter.php(825): App\Controllers\Auth->verifySign()
#5 /var/www/html/ci418/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#6 /var/www/html/ci418/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#7 /var/www/html/ci418/public/index.php(38): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/ci418/system/Commands/Server/rewrite.php(43): require_once('...')
#9 {main}
CRITICAL - 2022-02-19 01:40:57 --> CodeIgniter\BaseModel::transformDataToArray(): Return value must be of type array, string returned
#0 /var/www/html/ci418/system/BaseModel.php(704): CodeIgniter\BaseModel->transformDataToArray()
#1 /var/www/html/ci418/system/Model.php(625): CodeIgniter\BaseModel->insert()
#2 /var/www/html/ci418/system/BaseModel.php(658): CodeIgniter\Model->insert()
#3 /var/www/html/ci418/app/Controllers/Auth.php(122): CodeIgniter\BaseModel->save()
#4 /var/www/html/ci418/system/CodeIgniter.php(825): App\Controllers\Auth->verifySign()
#5 /var/www/html/ci418/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#6 /var/www/html/ci418/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#7 /var/www/html/ci418/public/index.php(38): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/ci418/system/Commands/Server/rewrite.php(43): require_once('...')
#9 {main}
CRITICAL - 2022-02-19 01:48:25 --> Call to undefined method CodeIgniter\HTTP\IncomingRequest::ggetPostGet()
#0 /var/www/html/ci418/system/CodeIgniter.php(825): App\Controllers\Auth->verifySign()
#1 /var/www/html/ci418/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#2 /var/www/html/ci418/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#3 /var/www/html/ci418/public/index.php(38): CodeIgniter\CodeIgniter->run()
#4 /var/www/html/ci418/system/Commands/Server/rewrite.php(43): require_once('...')
#5 {main}
CRITICAL - 2022-02-19 02:13:56 --> There is no data to update.
#0 /var/www/html/ci418/system/BaseModel.php(1574): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset()
#1 /var/www/html/ci418/system/BaseModel.php(832): CodeIgniter\BaseModel->transformDataToArray()
#2 /var/www/html/ci418/system/Model.php(651): CodeIgniter\BaseModel->update()
#3 /var/www/html/ci418/app/Controllers/Challenge.php(141): CodeIgniter\Model->update()
#4 /var/www/html/ci418/system/CodeIgniter.php(825): App\Controllers\Challenge->updateBasic()
#5 /var/www/html/ci418/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#6 /var/www/html/ci418/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#7 /var/www/html/ci418/public/index.php(38): CodeIgniter\CodeIgniter->run()
#8 /var/www/html/ci418/system/Commands/Server/rewrite.php(43): require_once('...')
#9 {main}
CRITICAL - 2022-02-19 14:37:34 --> syntax error, unexpected token "<"
#0 /var/www/html/ci418/system/View/View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#1 /var/www/html/ci418/system/Common.php(1089): CodeIgniter\View\View->render()
#2 /var/www/html/ci418/app/Controllers/Challenge.php(56): view()
#3 /var/www/html/ci418/system/CodeIgniter.php(825): App\Controllers\Challenge->register()
#4 /var/www/html/ci418/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#5 /var/www/html/ci418/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#6 /var/www/html/ci418/public/index.php(38): CodeIgniter\CodeIgniter->run()
#7 /var/www/html/ci418/system/Commands/Server/rewrite.php(43): require_once('...')
#8 {main}
CRITICAL - 2022-02-19 19:09:11 --> Undefined variable $tagline
#0 /var/www/html/ci418/app/Views/challenge/registration.php(46): CodeIgniter\Debug\Exceptions->errorHandler()
#1 /var/www/html/ci418/system/View/View.php(210): include('...')
#2 /var/www/html/ci418/system/View/View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 /var/www/html/ci418/system/Common.php(1089): CodeIgniter\View\View->render()
#4 /var/www/html/ci418/app/Controllers/Challenge.php(56): view()
#5 /var/www/html/ci418/system/CodeIgniter.php(825): App\Controllers\Challenge->register()
#6 /var/www/html/ci418/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#7 /var/www/html/ci418/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#8 /var/www/html/ci418/public/index.php(38): CodeIgniter\CodeIgniter->run()
#9 /var/www/html/ci418/system/Commands/Server/rewrite.php(43): require_once('...')
#10 {main}
CRITICAL - 2022-02-19 19:09:27 --> Undefined variable $tagline
#0 /var/www/html/ci418/app/Views/challenge/registration.php(46): CodeIgniter\Debug\Exceptions->errorHandler()
#1 /var/www/html/ci418/system/View/View.php(210): include('...')
#2 /var/www/html/ci418/system/View/View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 /var/www/html/ci418/system/Common.php(1089): CodeIgniter\View\View->render()
#4 /var/www/html/ci418/app/Controllers/Challenge.php(56): view()
#5 /var/www/html/ci418/system/CodeIgniter.php(825): App\Controllers\Challenge->register()
#6 /var/www/html/ci418/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#7 /var/www/html/ci418/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#8 /var/www/html/ci418/public/index.php(38): CodeIgniter\CodeIgniter->run()
#9 /var/www/html/ci418/system/Commands/Server/rewrite.php(43): require_once('...')
#10 {main}
CRITICAL - 2022-02-19 19:12:46 --> Undefined variable $tagline
#0 /var/www/html/ci418/app/Views/challenge/registration.php(57): CodeIgniter\Debug\Exceptions->errorHandler()
#1 /var/www/html/ci418/system/View/View.php(210): include('...')
#2 /var/www/html/ci418/system/View/View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 /var/www/html/ci418/system/Common.php(1089): CodeIgniter\View\View->render()
#4 /var/www/html/ci418/app/Controllers/Challenge.php(56): view()
#5 /var/www/html/ci418/system/CodeIgniter.php(825): App\Controllers\Challenge->register()
#6 /var/www/html/ci418/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#7 /var/www/html/ci418/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#8 /var/www/html/ci418/public/index.php(38): CodeIgniter\CodeIgniter->run()
#9 /var/www/html/ci418/system/Commands/Server/rewrite.php(43): require_once('...')
#10 {main}
CRITICAL - 2022-02-19 19:18:20 --> Undefined property: stdClass::$keyword
#0 /var/www/html/ci418/app/Views/challenge/registration.php(24): CodeIgniter\Debug\Exceptions->errorHandler()
#1 /var/www/html/ci418/system/View/View.php(210): include('...')
#2 /var/www/html/ci418/system/View/View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 /var/www/html/ci418/system/Common.php(1089): CodeIgniter\View\View->render()
#4 /var/www/html/ci418/app/Controllers/Challenge.php(130): view()
#5 /var/www/html/ci418/system/CodeIgniter.php(825): App\Controllers\Challenge->basicForm()
#6 /var/www/html/ci418/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#7 /var/www/html/ci418/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#8 /var/www/html/ci418/public/index.php(38): CodeIgniter\CodeIgniter->run()
#9 /var/www/html/ci418/system/Commands/Server/rewrite.php(43): require_once('...')
#10 {main}
CRITICAL - 2022-02-19 19:58:10 --> Undefined variable $city
#0 /var/www/html/ci418/app/Views/challenge/registration.php(115): CodeIgniter\Debug\Exceptions->errorHandler()
#1 /var/www/html/ci418/system/View/View.php(210): include('...')
#2 /var/www/html/ci418/system/View/View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 /var/www/html/ci418/system/Common.php(1089): CodeIgniter\View\View->render()
#4 /var/www/html/ci418/app/Controllers/Challenge.php(130): view()
#5 /var/www/html/ci418/system/CodeIgniter.php(825): App\Controllers\Challenge->basicForm()
#6 /var/www/html/ci418/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#7 /var/www/html/ci418/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#8 /var/www/html/ci418/public/index.php(38): CodeIgniter\CodeIgniter->run()
#9 /var/www/html/ci418/system/Commands/Server/rewrite.php(43): require_once('...')
#10 {main}
ERROR - 2022-02-19 23:25:08 --> Unknown column 'basicinfo.deleted_at' in 'where clause'
CRITICAL - 2022-02-19 23:25:08 --> Unknown column 'basicinfo.deleted_at' in 'where clause'
#0 /var/www/html/ci418/system/Database/MySQLi/Connection.php(292): mysqli->query()
#1 /var/www/html/ci418/system/Database/BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute()
#2 /var/www/html/ci418/system/Database/BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 /var/www/html/ci418/system/Database/BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query()
#4 /var/www/html/ci418/system/Model.php(145): CodeIgniter\Database\BaseBuilder->get()
#5 /var/www/html/ci418/system/BaseModel.php(523): CodeIgniter\Model->doFind()
#6 /var/www/html/ci418/app/Controllers/Challenge.php(173): CodeIgniter\BaseModel->find()
#7 /var/www/html/ci418/system/CodeIgniter.php(825): App\Controllers\Challenge->erase()
#8 /var/www/html/ci418/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#9 /var/www/html/ci418/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#10 /var/www/html/ci418/public/index.php(38): CodeIgniter\CodeIgniter->run()
#11 /var/www/html/ci418/system/Commands/Server/rewrite.php(43): require_once('...')
#12 {main}
ERROR - 2022-02-19 23:27:25 --> Unknown column 'basicinfo.deleted_at' in 'where clause'
CRITICAL - 2022-02-19 23:27:25 --> Unknown column 'basicinfo.deleted_at' in 'where clause'
#0 /var/www/html/ci418/system/Database/MySQLi/Connection.php(292): mysqli->query()
#1 /var/www/html/ci418/system/Database/BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute()
#2 /var/www/html/ci418/system/Database/BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 /var/www/html/ci418/system/Database/BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query()
#4 /var/www/html/ci418/system/Model.php(142): CodeIgniter\Database\BaseBuilder->get()
#5 /var/www/html/ci418/system/BaseModel.php(523): CodeIgniter\Model->doFind()
#6 /var/www/html/ci418/app/Controllers/Challenge.php(173): CodeIgniter\BaseModel->find()
#7 /var/www/html/ci418/system/CodeIgniter.php(825): App\Controllers\Challenge->erase()
#8 /var/www/html/ci418/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#9 /var/www/html/ci418/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#10 /var/www/html/ci418/public/index.php(38): CodeIgniter\CodeIgniter->run()
#11 /var/www/html/ci418/system/Commands/Server/rewrite.php(43): require_once('...')
#12 {main}
ERROR - 2022-02-19 23:27:36 --> Unknown column 'basicinfo.deleted_at' in 'where clause'
CRITICAL - 2022-02-19 23:27:36 --> Unknown column 'basicinfo.deleted_at' in 'where clause'
#0 /var/www/html/ci418/system/Database/MySQLi/Connection.php(292): mysqli->query()
#1 /var/www/html/ci418/system/Database/BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute()
#2 /var/www/html/ci418/system/Database/BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 /var/www/html/ci418/system/Database/BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query()
#4 /var/www/html/ci418/system/Model.php(142): CodeIgniter\Database\BaseBuilder->get()
#5 /var/www/html/ci418/system/BaseModel.php(523): CodeIgniter\Model->doFind()
#6 /var/www/html/ci418/app/Controllers/Challenge.php(173): CodeIgniter\BaseModel->find()
#7 /var/www/html/ci418/system/CodeIgniter.php(825): App\Controllers\Challenge->erase()
#8 /var/www/html/ci418/system/CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#9 /var/www/html/ci418/system/CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#10 /var/www/html/ci418/public/index.php(38): CodeIgniter\CodeIgniter->run()
#11 /var/www/html/ci418/system/Commands/Server/rewrite.php(43): require_once('...')
#12 {main}
